src/USER-MISC/fix_charge_regulation.cpp

```diff
@@ -0,0 +1,1335 @@
+/* ----------------------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   http://lammps.sandia.gov, Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+/* ----------------------------------------------------------------------
+   Contributing author: Tine Curk (tcurk5@gmail.com) and Jiaxing Yuan (yuanjiaxing123@hotmail.com)
+------------------------------------------------------------------------- */
+#include "fix_charge_regulation.h"
+#include <mpi.h>
```

Comment:
Please rearrange the include files to match our conventions (header for implementation, then LAMMPS headers sorted alphabetically, then system headers w/o the headers that are included by default) as outlined in https://github.com/lammps/lammps/blob/master/doc/include-file-conventions.md

See `src/MC/fix_widom.cpp` for example for a similar file that is fully compliant with those conventions.

Reply:
done.